source/containers.tex

```diff
@@ -366,10 +366,11 @@
 swap.
 
 \pnum
+\indextext{reversible container|see{container, reversible}}%
 If the iterator type of a container belongs to the bidirectional or
 random access iterator categories\iref{iterator.requirements},
 the container is called
-\term{reversible}
+\defnx{reversible}{container!reversible}
```

Comment:
Please use \defnadj{reversible}{container} and drop the "see" above (which comes for free now).

Reply:
Hmm, using \defnadj would result in "reversible container" in the running text, but here we only want the word "reversible" to appear in the running text. \defnadjx also seems unsuitable for the same reason.

Maybe there ought to be a macro for defining an adjective while only making the adjective itself (and not the noun) appear in the running text?

Reply:
Ah, you're right. But that doesn't apply to the second change, below.